app/views/pizzas/index.html.erb

```diff
@@ -0,0 +1,109 @@
+<%
+=begin%>
+  In another life I would have come back to this page and dried it up drastically -- no reason to have this repeated three times.
+<%
+=end%>
+<main>
+<h2 class="title">List of Pizzas</h2>
+
+<section class="media-table">
+  <h4>Thin Pizza</h4>
+  <table class="table">                         
+    <thead>
+      <tr>   
+        <th width="80">Votes</th>                       
+        <th>Name</th>
+        <th>Crust</th>
+        <th>Sauce</th>
+        <th>Cheese</th>
+        <th>Toppings</th>
+        <th>Created</th>
+        <th>Upvote</th>
+      </tr>
+    </thead>
+    </tbody>
+      <% Pizza.pizzas_by_crust("thin").each do |pizza| %>  
+          <tr>
+            <td><%= pizza.votes.count %></td>
+            <td><%= link_to pizza.name, pizza_path(pizza.id) %></td>
+            <td><%= pizza.crust %></td>  
+            <td><%= pizza.sauce %></td>  
+            <td><%= pizza.cheese %></td>  
+            <td><%= pizza.toppings %></td> 
+            <td><%= pizza.created_at.strftime("%m/%d/%Y") %></td> 
+            <td><%= button_to "Upvote Pizza", upvote_path(pizza.id), method: :post, class: 'btn btn-primary' %></td>
+          </tr>
+      <% end %>
+    </tbody>                     
+    </table>
+  </section>
+<section class="media-table">
+  <h4>Thick Pizza</h4>
+  <table class="table">                         
+    <thead>
+      <tr>   
+        <th width="80">Votes</th>                       
```

Comment:
You have the same code to show a list of works repeated 3 times. Could you use a view partial or a loop to DRY this up?